src/v2/view-builder/views/PollView.js

```diff
@@ -0,0 +1,35 @@
+import { loc } from 'okta';
+import BaseView from '../internals/BaseView';
+import BaseForm from '../internals/BaseForm';
+import polling from './shared/polling';
+
+const Body = BaseForm.extend(Object.assign(
+  {
+    title () {
+      return  loc('safemode.poll.title', 'login');
+    },
+
+    noButtonBar: true,
+    
+    initialize () {
+      BaseForm.prototype.initialize.apply(this, arguments);
+      this.startPolling();
```

Comment:
I looked at `startPolling`, it doesn't call the callback immediately but waits for {refresh} ms to do the first call. Kind of a waste, unless it's unlikely the first call will ever be successful?

Reply:
That's intentional.. If any remediation resolves to `poll` means server asks sign-widget to wait for X seconds before trying out the same request again.